src/v2/ion/i18nTransformer.js

```diff
@@ -63,7 +63,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'select-authenticator-enroll.authenticator.webauthn': 'oie.webauthn.label',
   'select-authenticator-enroll.authenticator.security_question': 'oie.security.question.label',
   'select-authenticator-enroll.authenticator.okta_verify': 'oie.okta_verify.label',
-  'select-authenticator-enroll.authenticator.google_authenticator': 'oie.google_authenticator.label',
+  'select-authenticator-enroll.authenticator.google_otp': 'oie.google_authenticator.label',
```

Comment:
do we have to update the properties on the right side? 
probably not since we're mapping them to the left side which is fixed, right?

Reply:
no. because we try to reuse the property exists in `login.properites` on the right side

Reply:
No, we don't have to update the entries on the right, since they are keys that we are still using in login.properties.